server/src/main/resources/db/migration/V20240424114300__add_sha256_to_public_file_versions.sql

```diff
@@ -0,0 +1,4 @@
+/*
+** add `checksum` column to public_files_table
+*/
+ALTER TABLE public_file_versions add COLUMN sha256 varchar(255);
```

Comment:
fwiw I think [PostgreSQL recommends](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_default) just using the `TEXT` column type unless you explicitly want the db to throw if the value is over the character limit